src/bp/ui-studio/src/web/components/ChatEmulator/Emulator.jsx

```diff
@@ -225,11 +262,19 @@ export default class EmulatorChat extends React.Component {
     this.setState({ isTypingHidden: !this.state.isTypingHidden })
   }
 
+  hideSettings = () => this.setState({ isSettingsOpen: false })
+  displaySettings = () => this.setState({ isSettingsOpen: true })
+  updateSettings = newSettings => this.setState({ ...newSettings })
+
+  toggleRawPayload = () => this.setState({ isSendingRawPayload: !this.state.isSendingRawPayload })
```

Comment:
How sending a raw payload is in the scope of authenticatedConversation ?

Reply:
It was used for debugging while working on that, so I merged it with my current branch